configure.ac

```diff
@@ -30,7 +30,7 @@ AC_CONFIG_HEADERS([config.h:config.in])dnl Keep filename to 8.3 for MS-DOS.
 AC_CONFIG_SRCDIR([libdrizzle/drizzle.cc])
 
 #shared library versioning
-LIBDRIZZLE_LIBRARY_VERSION=10:0:0
+LIBDRIZZLE_LIBRARY_VERSION=10:1:0
```

Comment:
Because of the documentation here: https://github.com/andreas-bok-sociomantic/libdrizzle-redux/blob/27af5a107f880b76156a317a490f3c2d1a82344d/configure.ac#L33

There is an issue to change this though here: https://github.com/sociomantic-tsunami/libdrizzle-redux/issues/92